aptos-move/aptos-vm/src/block_executor/vm_wrapper.rs

```diff
@@ -61,23 +60,15 @@ impl<'a, S: 'a + StateView + Sync> ExecutorTask for AptosExecutorTask<'a, S> {
         view: &impl StateView,
         txn: &PreprocessedTransaction,
         txn_idx: TxnIndex,
-        materialize_deltas: bool,
+        _aggregator_enabled: bool,
     ) -> ExecutionStatus<AptosTransactionOutput, VMStatus> {
         let log_context = AdapterLogSchema::new(self.base_view.id(), txn_idx as usize);
 
         match self
             .vm
             .execute_single_transaction(txn, &view.as_move_resolver(), &log_context)
```

Comment:
hm, but we would have to propagate this to the execution and set the flag in there to materialize the aggregators, right? Otherwise the sequential execution should break currently, as it would still return deltas but nobody would materialize the deltas anymore?

Reply:
Done.